app/models/organization.rb

```diff
@@ -78,6 +79,16 @@ def payment_provider(provider)
     end
   end
 
+  # TODO: uncomment these and fix all the specs
+  # def webhook_url?
+  #   webhook_endpoints.any?
+  # end
+
+  # # TODO: deprecated
+  # def webhook_url
+  #   webhook_endpoints&.first&.webhook_url
+  # end
```

Comment:
Do we still need this?

Reply:
Not anymore, removing it and changing all the calls from `organization.webhook_url?` to `organization.webhook_endpoints.any?`